cmd/av/stack.go

```diff
@@ -0,0 +1,173 @@
+package main
+
+import (
+	"emperror.dev/errors"
+	"github.com/spf13/cobra"
+	"strconv"
+)
+
+var stackCmd = &cobra.Command{
+	Use:   "stack",
+	Short: "managed stacked pull requests",
+}
+
+var stackBranchFlags struct {
+	// The parent branch to base the new branch off.
+	// By default, this is the current branch.
+	Parent string
+}
+var stackBranchCmd = &cobra.Command{
+	Use:   "branch <branch name>",
+	Short: "create a new stacked branch",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		if len(args) != 1 {
+			return errors.New("branch name is required")
+		}
+		return errors.New("unimplemented")
+	},
+}
+
+var stackSyncFlags struct {
+	// Set the parent of the current branch to this branch.
+	// This effectively re-roots the stack on a new parent.
+	Parent string
```

Comment:
necessary?

Reply:
Yeah. This is how you'd be able to tell Aviator that you want to change your branch to be based on something else. So if you do some work then realize you need to actually make it into a stacked PR, you can commit and run `av stack sync --parent my-feature`.

Reply:
Makes sense. Perhaps then the comment should be more describing this as adding the branch to the stack.
A secondary behavior is that this can be also used to move the branch somewhere else in stack, but that's the impression I got reading the comment.